client/src/components/home/footer.tsx

```diff
@@ -0,0 +1,72 @@
+import Link from 'next/link';
+
+import { ExternalLink } from 'lucide-react';
+import EuropeFlag from 'public/images/europe-flag.svg';
+import LinkedinIcon from 'public/images/linkedin-icon.svg';
+import MailIcon from 'public/images/mail-icon.svg';
+import XIcon from 'public/images/x-icon.svg';
+
+/* eslint-disable @next/next/no-img-element */
+const Footer = () => (
+  <div id="about">
+    <div className="flex w-full flex-col items-center justify-center gap-[20px] bg-gray-850 px-10 py-[74px]">
+      <Link
+        className="flex items-center justify-center gap-2.5"
+        href="https://irc-orcasa.eu/"
+        target="_blank"
+      >
+        <div className="font-serif text-lg leading-[42px] text-green-700">
+          Impact4Soil has been created thanks to the ORCaSa project
+        </div>
+        <ExternalLink className="relative h-6 w-6 text-green-700" />
+      </Link>
+      <Link href="https://irc-orcasa.eu/" target="_blank">
+        <img
+          className="h-[77px] w-[255px]"
+          src="/images/orcasa-logo-with-text.png"
+          alt="Orcasa logo"
+        />
+      </Link>
```

Comment:
1. You should use the `Image` component, no?
2. Orcasa is spelled ORCaSa
3. I would add a span: `<span className="sr-only">ORCaSa</span>